src/pages/ecosystem/contributing.md

```diff
@@ -46,8 +46,8 @@ When workin locally with the site, a few things are needed:
 
 All of the code for this site is open source, located at the [GitHub repository here](https://github.com/blockstack/docs).
 Before you start editing anything, you will need to fork the repo so that you can have your own copy of the code under
-your GitHub profile. On the [repository's page](https://github.com/blockstack/docs), you should be able to
-see a button in the upper right of the screen that says "Fork". [You can read about Forking here.](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
+your GitHub profile. On the [repository's page](https://github.com/blockstack/docs.blockstack), you should be able to
+see a button in the upper right of the screen that says "Fork." [You can read about Forking here.](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
```

Comment:
Wrong change

Reply:
this is actually according to the rules. if the quote is at the end of the sentence, the comma or dot should be inside the quote. Another "controversial" guideline. I'd just follow it for now and work on our own guidelines later